GDD.md

```diff
@@ -11,25 +15,50 @@ This runs on Windows, Linux, and in Web.
 
 ## Development Software
 Programming: Godot Engine
+
 Art: 
+
 Music: 
 
 ## Genre
-Singleplayer, [genre2], [genre3]
+Singleplayer Metroidvania Turn-based Tactics
 
 ## Target Audience
 
 
 # Concept
 ## Gameplay Overview
+Navigate the overworld, armed with the Potion Launcher.
+The overworld is made up of multiple rooms. When player enters a new room, the room is shrouded in SHADOW (room is grayscale), and filled with SHADOW creatures.
+The ALCHEMIST must purify the room without hurting the SHADOW CREATURES, by getting to the SUNflowers (colored).
+Upon reaching enough SUNflowers, the ALCHEMIST's SUNSPARK activates, purifying the room and every SHADOW creatures within it.
 
 ## Theme Interpretations
-Shadow: 
-Alchemy: 
+Shadow: SHADOW has descended upon the world. Player attempts to restore the sun to banish the SHADOW.
 
-## Primary Mechanics
+Alchemy: Player is an ALCHEMIST, who collects reagents and brews new potions that interact with the environment to address challenges. 
```

Comment:
should have more words here imo. that is maybe make it a little bit more abstract